.chloggen/pkg-batchperresourceattr.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: batchperresourceattr
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Don't reset the resources instances, in case the caller wants to reuse it.
```

Comment:
```suggestion
Fix setting the resource attributes on all the resource that are part of the batch, by copying the original resource attributes instead of moving them.
```